lib/config/validation.ts

```diff
@@ -269,6 +270,18 @@ export async function validateConfig(
             if (key === 'extends') {
               for (const subval of val) {
                 if (is.string(subval)) {
+                  try {
+                    await getPreset(subval, config);
+                  } catch (err) {
+                    logger.warn(
+                      { err, preset: subval },
+                      `Could not resolve preset during config validation`
+                    );
```

Comment:
```suggestion
                    logger.debug(
                      { err, preset: subval },
                      `Could not resolve preset during config validation`
                    );
```

i think we don't need to warn here if we pass an error below?